solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -59,8 +63,10 @@
   public static final ImmutableList<String> KEYWORD_TYPES = ImmutableList.<String>builder()
     .add(SolidityKeywords.getKeyowrdTypes()).build();
 
-  public SoliditySensor(FileLinesContextFactory fileLinesContextFactory) {
-    this.fileLinesContextFactory = fileLinesContextFactory;
+  public SoliditySensor(CheckFactory checkFactory, FileLinesContextFactory fileLinesContextFactory) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "fileLinesContextFactory". ([squid:S1172](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=24&issues=AWRgb64yl6vpliCe2LN0&open=AWRgb64yl6vpliCe2LN0)

[ISSUE_KEY]: AWRgb64yl6vpliCe2LN0